src/Rings.jl

```diff
@@ -81,6 +81,17 @@ end
 
 *(x::RingElement, y::RingElem) = parent(y)(x)*y
 
+
+"""
+    divexact(x, y)
+
+Return the exact quotient of `x` by `y`, i.e. an element
```

Comment:
Actually, the quotient may not be unique (e.g. in Z/6Z,  divexact(4,2) could return either 2 or 5. So perhaps:
```suggestion
Return an exact quotient of `x` by `y`, i.e. an element
```

Reply:
Good point!